widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementRepository.java

```diff
@@ -68,13 +58,14 @@ public void unregisterCallVisualizerEngagementListener(Consumer<OmnibrowseEngage
     }
 
     public boolean hasOngoingEngagement() {
-        return gliaCore.getCurrentEngagement().isPresent();
+        return gliaCore.getCurrentEngagement().isPresent();//TODO when ending engagement locally this shit should be reworked,
+        // because in fact, we're clearing state before we will receive engagement end event, end this could cause bubble to show up after engagement end
```

Comment:
Just to double check, this TODO is still needed, right?

Reply:
Yes)